packages/fxa-auth-server/scripts/audit-tokens.ts

```diff
@@ -409,29 +350,13 @@ async function auditAll() {
       );
     }
 
-    // Rows orphaned by missing oauth client
-    set = [
-      tables.clientDevelopers,
-      tables.tokens,
-      tables.codes,
-      tables.refreshTokens,
-    ];
-    for (const table of set) {
-      await auditOrphanedRows(
-        { name: table, keyCol: 'clientId' },
-        { name: tables.clients, keyCol: 'id' }
-      );
-    }
-
-    // Rows orphaned by missing developer
-    set = [tables.clientDevelopers];
+    set = [tables.devices];
```

Comment:
Add one more fxa audit devices > sessionTokens, and phase out the device table specific audit which joined on an fxa_oauth table.